utils/db.py

```
@@ -135,3 +135,40 @@ def get_match_id_by_teams_and_date(home_team_id: int, away_team_id: int, match_d
+                return rows
+    except Exception as e:
+        logging.error(f"투수 선수 정보 조회 실패: {e}")
+        return None
+
+def get_hitters_by_team_id(team_id: int):
+    connection = get_connection()
+    if not connection:
+        return None
+    try:
+        with connection:
+            with connection.cursor() as cursor:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)